src/js/hooks/cycleCount/useResolveStep.js

```diff
@@ -125,13 +125,30 @@ const useResolveStep = () => {
     });
   };
 
+  const moveCustomItemsToTheBottom = (cycleCountItems) => {
```

Comment:
Can you add a simple comment on why this is here? (something about that we want to keep ordering of items that was before a save action, to display the validation properly, and that when we come back to it, then we don't care about this ordering)